ouroboros-consensus-test/test-consensus/Test/Consensus/MiniProtocol/ChainSync/Client.hs

```diff
@@ -669,15 +716,21 @@ getChainUpdates = reverse . cusUpdates
 -- '_currentChain'.
 prop_genChainUpdates :: SecurityParam -> Int -> Property
 prop_genChainUpdates securityParam n =
-    forAll (genChainUpdates securityParam n emptyUpdateState) $ \cus ->
+    forAll (genChainUpdates SelectedChainBehavior securityParam n emptyUpdateState) $ \cus ->
       Chain.applyChainUpdates (toChainUpdates (getChainUpdates cus)) Genesis ===
       Just (cusCurrentChain cus)
 
-genChainUpdates :: SecurityParam
+data UpdateBehavior =
```

Comment:
I anticipate someone later will benefit from a bit of Haddock on this type/these constructors 😄